app/main.py

```diff
@@ -302,15 +302,9 @@ def update_live_alerts_data(
 
     # Then, we construct a dictionary whose keys are the event IDs (as integers) and values are the corresponding
     # "is_acknowledged" field in the events table (boolean)
-    is_event_acknowledged = {}
-    for event in all_events:
-        is_event_acknowledged[event["id"]] = event["is_acknowledged"]
-
-    # We map this dictionary upon the column and revert the booleans with ~ as we want unacknowledged events
-    mask_acknowledgement = ~all_alerts["event_id"].map(is_event_acknowledged)
-
```

Comment:
This is no longer useful regarding incoming [PR](https://github.com/pyronear/pyro-platform/pull/73) 

You changed it because it's raising an error right ?